assets/ui.css

```diff
@@ -231,7 +243,6 @@
   background: #fff;
   border: 5px solid #666;
   min-width: 20%;
-  width: 79%;
```

Comment:
Why was this removed? I'm seeing modal like this now:
![image](https://user-images.githubusercontent.com/8426945/72170510-03af9a00-33f7-11ea-860d-33784c9f9378.png)

Reply:
There was a small gap between the heading and the boundary in some of the modals(search definitions). I wanted to remove that actually. Correcting this!